PYTHON2/day20/sday20.py

```diff
@@ -0,0 +1,11 @@
+import re
+str=raw_input()
```

Comment:
That's not true: you are forcing `str` (which is a type) to be an actual string. Try to call for example `print str(2)` to see what happens